lib/UserDiscoveryModules/ManualUserMapping.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * @copyright Copyright (c) 2018 Bjoern Schiessle <bjoern@schiessle.org>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\GlobalSiteSelector\UserDiscoveryModules;
+
+/**
+ * Class ManualUserMapping
+ *
+ * allows you to manage a local file which maps a arbitrary SAML parameter to
+ * a initial location of the user.
+ *
+ * Therefore you have to define to values in the config.php
+ *
+ * 'gss.discovery.manual.mapping.file' => '/path/to/file'
+ * 'gss.discovery.manual.mapping.parameter' => 'idp-parameter'
```

Comment:
I would like to have this as well in the README with a general overview of all the available modules and their config.